tests/conftest.py

```diff
@@ -1,8 +1,15 @@
 import django
+import pytest
 
 
 def pytest_configure():
     from django.conf import settings
 
     settings.configure()
     django.setup()
+
+
+@pytest.fixture
+def clear_resolvers():
+    from winter.argument_resolver import _resolvers
```

Comment:
I think we shouldn't import private variable
may be add function for clearing  resolvers?

Reply:
it was a starting point for discussion, I didn't want to merge it